src/api/likesApi.js

```diff
@@ -0,0 +1,11 @@
+import { APP_ID } from '../config.js';
+
+const baseUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${APP_ID}/likes`;
+
+export const addLike = (itemId) => fetch(baseUrl, {
+  method: 'POST',
+  headers: { 'Content-Type': 'application/json' },
+  body: JSON.stringify({ item_id: itemId }),
+}).then((response) => response.ok);
+
+export const getLikes = () => fetch(baseUrl).then((response) => response.json());
```

Comment:
- [ ]  OPTIONAL
Great job with implementations and design so far! However, I think you can improve upon the UX of your _likes_ features in these ways:
    - Like icon should toggle to indicate _like_ or _unlike_ when a particular user clicks on the like icon. If the page is refreshed, then assume that new user is present.
    - Total count of likes per user/site visitor should be 1. So, let's assume a user clicks a like, and the number of likes for a particular item should be `+1`, and if the same user clicks the like again the number of likes for that same object should be `-1`. If the page is refreshed, then assume that new user is present. 😉